Chapter2/Exercise10/Palindromo.js

```diff
@@ -1,6 +1,13 @@
 //Escreve uma função que verifique se uma palavra lida de trás para frente é igual à leitura normal.
 
-function Palindromo(s) {}
+function Palindromo(s) {
+    const reversed = s.split("").reverse().join("")
+    if (reversed == s) {
```

Comment:
Great! Just a little warning, in JavaScript we have two operators for comparison: `==` and `===`. While the first tries to convert both arguments to the same type (for example: `'5' == 5` would be **true**), the last **strictly** compares both items, meaning they have to have the same value (for example: `'5' === 5` would be **false**, but `5 === 5` would be **true**). Usually, we should always use `===` to avoid problems, unless we explicitly want to compare values not strictly.

In this case it would work either way, but since we know we are just comparing strings, there's no need to use `==`.

On a final note, we could make this `if ... else ...` into just one line, like so: `return reversed === s` ;)

Reply:
Uhhh, great advice.
I'll make sure I'll get it right next time :)